src/IF.Lastfm.Core.Tests.Integration/Commands/UserGetWeeklyChartTests.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace IF.Lastfm.Core.Tests.Integration.Commands
+{
+    public class UserGetWeeklyChartTests : CommandIntegrationTestsBase
+    {
+        [Test]
+        public async Task GetChartList_Success()
+        {
+            var response = await Lastfm.User.GetWeeklyChartListAsync(INTEGRATION_TEST_USER);
+            var chartlist = response.Content;
+            
+            Assert.IsTrue(response.Success);
+            Assert.IsNotEmpty(chartlist);
+            Assert.AreEqual(typeof(double), chartlist.First().To.GetType());
```

Comment:
The type assertions here are unneccessary, since it's not possible for the types to be incorrect - the command files wouldn't compile.